packages/next/src/server/lib/incremental-cache/index.ts

```diff
@@ -121,13 +129,19 @@ export class IncrementalCache implements IncrementalCacheType {
     const debug = !!process.env.NEXT_PRIVATE_DEBUG_CACHE
     this.hasCustomCacheHandler = Boolean(CurCacheHandler)
     if (!CurCacheHandler) {
-      if (fs && serverDistDir) {
+      const _globalThis: any = globalThis
+      const globalCacheHandler =
```

Comment:
should this be part of https://github.com/vercel/next.js/blob/canary/packages/next/src/server/after/builtin-request-context.ts ?

Reply:
Oh yea we can just re-use